src/cdrafter.cc

```diff
@@ -35,7 +35,13 @@ SC_API int drafter_c_parse(const char* source,
 
     if (result) {
         std::stringstream resultStream;
-        serializer.process(drafter::WrapResult(blueprint, options), resultStream);
+        try {
+            serializer.process(drafter::WrapResult(blueprint, options), resultStream);
+        } 
+        catch (std::exception e) {
+            std::cerr << "Error: " << e.what() << std::endl;
```

Comment:
Instead of printing the error why are we not adding it to `blueprint.report.error.message`? Similarly below.